src/Chapter2.hs

```diff
@@ -39,6 +39,7 @@ Now, if you are ready, bring it on!
 -}
 
 module Chapter2 where
+import Data.List (take, drop)
```

Comment:
These functions come with `Prelude` module, so no need to import them manually from `Data.List` 🙂